### PR TITLE
Contests list

### PR DESCRIPTION
**Pull request by neSpecc**
- [x] @demyashev 
- [x] @loopdigga98 
- [x] @n0str 
- [x] @illiiiillllilii 
- [x] @PolinaShneider 
- [x] @spanic 
- [x] @ivanszhuravlev 
- [x] @Kapitonova

### CODE REVIEW

**Comment by neSpecc** [comment]
тестировать можно на http://stage.ifmo.su/contests

**Comment by loopdigga98** [comment]
screenshot_2016-03-03-21-38-31

File: migrations/2016_03_03_Contests.sql [hunk 1]
```diff
@@ -59,4 +59,6 @@ ALTER TABLE `Contests`
 -- AUTO_INCREMENT for table `Contests`
 --
 ALTER TABLE `Contests`
-MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=3;
\ No newline at end of file
+MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=3;
+
```
Comment (demyashev) [inline migrations/2016_03_03_Contests.sql]: Странный diff

**Comment by neSpecc** [comment]
@loopdigga98 поправил

**Pull request merged**